src/config/services/configuring.md

```diff
@@ -5,28 +5,14 @@ options can be set via a `conf` file in the service directory. This allows
 service customization without modifying the service directory provided by the
 relevant package.
 
+Check the service file for how to pass configuration parameters, but services
+usually expect `OPTS="--value1 ..."` as values.
+
 To make more complex customizations than provided by default, edit the service.
 
 ## Editing services
 
-To edit a service, first duplicate its service directory under a different name:
-
-```
-# cp -aR /etc/sv/service_name /etc/sv/service_name_edited
-```
-
-The contents of the duplicated directory can then be edited as needed, for
-example, by using `vi(1)`:
-
-```
-# vi /etc/sv/service_name_edited
-```
-
-After editing, the old service should then be stopped, disabled and replaced
-with the new one:
-
-```
-# sv down service_name
-# rm /var/service/service_name
-# ln -s /etc/sv/service_name_edited /var/service/
-```
+To edit a service, first copy its service directory to a different directory
+name, otherwise `xbps` will overwrite the service directory. Then, edit the new
```

Comment:
Suggest

```
otherwise [xbps-install(1)] (man page link) might overwrite your changes.
```

Reply:
good catch

Reply:
btw the man pages on voidlinux.org are outdated. The websitel says September 20,2016, while the local version is March 3, 2020.

Reply:
They are outdated since xbps switched to zstd. I changed the little bits necessary in the tool that extracted manuals so it can work with zstd just yesterday, and we're looking to put it into the infrastructure. Software like iwd doesn't even have manpages there right now.